### ISSUE TITLE
Make default preferences for Backup, FSEQ, RenderCache and Sequences relative to the show folder

### ISSUE THREAD

**Issue by derwin12** [xLightsSequencer/xLights#4302]
As it is now, when folks copy and start their next show or perhaps copy & split into a christmas and halloween their preferences are locked into the prior show folder. By saving relative file names, it can, with validation, switch based on the show folder directory.

Request is to select and save just the relative portion. Perhaps showing the whole directory? but change if the show directory changes.

**Comment by dkulp** [comment]
I'm a bit against this for fseq and rendercache.   I don't have those directories anywhere close to where my show folders are and I really do NOT want them to be relative to the show folder.

**Comment by derwin12** [comment]
> I'm a bit against this for fseq and rendercache. I don't have those directories anywhere close to where my show folders are and I really do NOT want them to be relative to the show folder.

Agreed .. but if the folder does include the current show folder - then make it relative? Just a thought. Or perhaps a checkbox?

**Comment by derwin12** [comment]
Q: where does fpp connect look for the fseq files - obviously in and optionally under the show folder. Does it also check the fseq setting recorded in xlights?

**Comment by TripleJ3EB** [comment]
> Agreed .. but if the folder does include the current show folder - then make it relative?

I would love this with anything path related in xlights, if its within the show folder when its originally saved, make it relative.  this would be very helpful with effect reference locations (image/video/etc) when you use multiple computers to work on sequences.
